templates/actors/actor-combat.html

```diff
@@ -47,7 +47,7 @@
         {{/if}}
 
         {{#if item.Qualities.length}}
-        <span class="label quality">Q:</span>
+        <span class="label quality">{{localize "Q:"}}</span>
```

Comment:
I don't see  the corresponding keys for this (or the ones below it). I feel like the key should be more specific too.